c7n/resources/redshift.py

```diff
@@ -557,6 +557,19 @@ class resource_type(object):
         date = 'SnapshotCreateTime'
 
 
+@actions.register('modify-security-groups')
+class RedshiftModifyGroups(ModifyGroupsAction):
+    """Modify security groups on an Redshift cluster"""
```

Comment:
same as rds re security groups in n action class name per disambiguation on subnet groups.